pkg/kvstore/etcd.go

```diff
@@ -198,34 +205,6 @@ func (e *etcdClient) renewSession() error {
 
 	go e.checkMinVersion()
 
-	e.renewLease()
-
-	return nil
-}
-
-func (e *etcdClient) renewLease() error {
-	if e.lease != nil {
-		e.client.Revoke(ctx.TODO(), e.lease.ID)
-	}
-
-	lease, err := e.client.Grant(ctx.TODO(), int64(LeaseTTL.Seconds()))
```

Comment:
It's a great idea to use the standard lease of the session but then we should call `NewSession()` with `func WithTTL(ttl int) SessionOption {` and pass in `LeaseTTL.Seconds()`